scripts/docker-integration-tests/aggregator/m3aggregator.yml

```diff
@@ -177,8 +177,7 @@ aggregator:
         maxJitterPercent: 0.5
       - flushInterval: 1h
         maxJitterPercent: 0.25
-    numWorkersPerCPU: 0.5
-    flushTimesPersistEvery: 10s
```

Comment:
i think we might want to keep the persist every. potentially just allow 0 or add an additional bool to indicate you want to always sync.

Reply:
Yeah, good call, will do

Reply:
Just gonna leave it out of these tests though, since I think we should be rolling forward with a zero `flushTimesPersistEvery` by default going forwards

Reply:
Also probably better to not break current configs